web/views/respondent_view.ex

```diff
@@ -1,8 +1,11 @@
 defmodule Ask.RespondentView do
   use Ask.Web, :view
 
-  def render("index.json", %{respondents: respondents, respondents_count: respondents_count}) do
-    %{data: %{respondents: render_many(respondents, Ask.RespondentView, "respondent.json")}, meta: %{count: respondents_count}}
+  def render("index.json", %{respondents: respondents} = render_index_data) do
+    %{
+      data: %{respondents: render_many(respondents, Ask.RespondentView, "respondent.json")},
+      meta: render_index_meta(render_index_data)
```

Comment:
Much better now. Easier to understand what is being rendered 🎉

Reply:
Totally! Thank you!!!